cmake/Modules/Findrxcpp.cmake

```diff
@@ -9,7 +9,7 @@ find_package_handle_standard_args(rxcpp DEFAULT_MSG
 
 
 set(URL https://github.com/Reactive-Extensions/rxcpp.git)
-set(VERSION 1b2e0589f19cb34d8cd58803677701dcf2161876)
+set(VERSION a7d5856385f126e874db6010d9dbfd37290c61de)
```

Comment:
Please add either a version comment, or why this particular commit is used (if it has no tag)

Reply:
Also would be nice to add version (if any) to other places, e.g dockerfiles

Reply:
Comments, yeah